src/superface-test.ts

```diff
@@ -155,6 +159,33 @@ export class SuperfaceTest {
     throw new UnexpectedError('Unexpected result object');
   }
 
+  private async replaceUnsupportedRecording(): Promise<void> {
+    const pathToUnsupported = this.composeRecordingPath('unsupported');
+    const pathToCurrent = this.composeRecordingPath();
+
+    await mkdirQuiet(joinPath(dirname(pathToCurrent), 'old'));
+
+    // TODO: compose version based on executed usecase
+    await rename(pathToCurrent, this.composeRecordingPath('1.0.0'));
+    await rename(pathToUnsupported, pathToCurrent);
+  }
+
+  private async canReplaceRecording(): Promise<boolean> {
+    const replaceRecording = parsePublishEnv(
+      process.env.PUBLISH_UNSUPPORTED_RECORDINGS
+    );
+
+    if (replaceRecording) {
+      if (!(await exists(this.composeRecordingPath('unsupported')))) {
+        return false;
+      }
+
+      return true;
```

Comment:
suggestion: Prefer positive condition

```suggestion
      if (await exists(this.composeRecordingPath('unsupported'))) {
        return true;
      }

      return false;
```